code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -40,7 +40,12 @@
 /obj/item/clothing/mask/facehugger/attackby(obj/item/O, mob/user, params)
 	return O.attack_obj(src, user, params)
 
-/obj/item/clothing/mask/facehugger/attack_alien(mob/user) //can be picked up by aliens
+/obj/item/clothing/mask/facehugger/attack_alien(mob/user) //can be picked up by aliens with the variable can_grab_facehuggers set to true
+	if(istype(user, /mob/living/carbon/alien/humanoid))
+		var/mob/living/carbon/alien/humanoid/alium = user
+		if(!alium.can_grab_facehuggers)
+			to_chat(user, "<span class='warning'>Your claws are too sharp to handle your unborn sisters!</span>")
+			return
```

Comment:
I think you mean `var/mob/living/carbon/alien/humanoid/alien` and not `alium`

Reply:
Oh, yeah, derp. It appears to work properly though, I forgot to change this. Saw it a while ago when I asked for help on discord and it left my mind. Will change and make sure everything is working properly.